src/sentry/utils/sdk_crashes/event_stripper.py

```diff
@@ -179,7 +179,7 @@ def strip_frame(frame: MutableMapping[str, Any]) -> MutableMapping[str, Any]:
             # The path field usually contains the name of the application, which we can't keep.
             for field in sdk_crash_detector.fields_containing_paths:
                 if frame.get(field):
-                    frame[field] = sdk_crash_detector.replace_sdk_frame_path(field)
+                    frame[field] = sdk_crash_detector.replace_sdk_frame_path(frame.get(field))
```

Comment:
How does this work? `replace_sdk_frame_path` previously accepted a field name, but now it accepts a field value?

Reply:
Will do this in an extra PR. Good catch.